build/py_gcs_snippets_menu_config.yml

```diff
@@ -1,9 +1,9 @@
-Google Bucket snippets:
+All of Us Python and Cloud Storage snippets:
```

Comment:
nit: filename gcs --> storage as well? Here and for R (and references to each of these files)